src/codegen/generators/llvm.rs

```diff
@@ -17,6 +17,43 @@ pub struct Llvm<'a> {
     pub builder: Builder<'a>,
 }
 
+pub trait GlobalValueExt {
+    fn into_constant(self) -> Self;
```

Comment:
I like the builder pattern, I dont particualry like the word "into" here. Maybe I got this wrong but I thought that the Into-Methods used in the rust std crates usually really transform something and return something different

do rust-builders usually use the word "into" when setting some fields?

Reply:
yes, true they convert to other types.
Suggetions? as_constant? make_constant?

Reply:
make_constant